### DISCUSSION TITLE
A11y project

### DISCUSSION THREAD

**Discussion by laurenchamps** [codelikeagirlau/accessibility-resources#5] [General]
Hey all, have you seen/used the [A11y project](https://www.a11yproject.com/)? 
I came across it a few weeks ago when I was looking into accessibility and maybe this is something you're already all familiar with? 
I haven't done a deep dive into it but it looks like a fantastic resource and they have a [checklist](https://www.a11yproject.com/checklist/) that they've adapted from the WCAG. Is this the type of thing you were thinking of, @lulen11 ?

**Comment by sarahesbie** [comment]
Oooh this is fabulous. Thanks for sharing @laurenchamps !

**Comment by lulen11** [comment]
Hey this is amazing @laurenchamps !! The checklist is just like what I was imagining!! We could definitely use this as inspiration <3
